### ISSUE TITLE
Display Error, data that conatins square bracket [ ],

### ISSUE THREAD

**Issue by phforest-wow** [MarkyMan4/filequery#51]
CMD:`filequery --filename test.parquet --query 'select * from 'test';`
I get this error message:
`rich.errors.MarkupError: closing tag '[/HWUVIDEO]' at position 1744 doesn't match any open tag`
Note: that I do not have any [ ] bracket in my file and that `HWUVIDEO` is not in my data.

I assume the rich console is causing this exception while trying to format the table.
Hard to attach my parquet file, as it is almost 1Gb, I've convert it to a csv file and that worked fine, (strange)
 
regards
Philippe

**Comment by MarkyMan4** [comment]
That is strange if `[\HWUVIDEO]` is not anywhere in your data. Hard to tell what's going on for sure without some sample data, but this does bring up a good point that any data needs to be escaped when formatting with a rich table. 

Looking at the rich docs, I found out there is a function available for escaping strings. I tried it out with some test data that contains square brackets and it seems to work properly. I'll include that in the next release and then I'd be curious to see if you are still having the issue when you query your parquet file.
